src/main/io/serial_msp.c

```diff
@@ -1290,7 +1293,16 @@ static bool processOutCommand(uint8_t cmdMSP)
         serialize8(masterConfig.baro_hardware);
         serialize8(masterConfig.mag_hardware);
         break;
-
+    case MSP_SPM_VTX_CONFIG:
```

Comment:
moreover, this is not specific to Spektrum. #2114 and follow ups are taking care of that.